### DISCUSSION TITLE
Hover menus

### DISCUSSION THREAD

**Discussion by nowhinjing-ever** [livewire/flux#694] [Feature Requests] [answered]
Is it possible to construct hover menus in flux? A hover menu means that when you hover over an item, the next level options are displayed, hovering over any of these items would display the next level and so on... You then click on an item to go to the required page. moving out of the area means that that lever disappears.

If not, could you add this to the list of to-dos please.

**Comment by mauritskorse** [comment, marked as answer]
Something like submenus? https://fluxui.dev/components/dropdown#submenus

I guess you can do this too within a navlist or navbar:
```
<flux:navlist>
        <flux:navlist.item href="#">Dashboard</flux:navlist.item>
        <flux:navlist.item href="#">Transactions</flux:navlist.item>

        <flux:dropdown position="right" align="start">
            <flux:navlist.item icon-trailing="chevron-down">Account</flux:navlist.item>

            <flux:menu>
                <flux:menu.item href="#">Profile</flux:menu.item>
                <flux:menu.item href="#">Settings</flux:menu.item>
                <flux:menu.item href="#">Billing</flux:menu.item>
                <flux:menu.submenu heading="Filter">
                    <flux:menu.item>Draft</flux:menu.checkbox>
                    <flux:menu.item>Published</flux:menu.checkbox>
                    <flux:menu.item>Archived</flux:menu.checkbox>
                </flux:menu.submenu>
            </flux:menu>
        </flux:dropdown>
    </flux:navlist>
```
![image](https://github.com/user-attachments/assets/65af0ee4-df16-4ab3-8ca5-043d3d62b927)

**Reply by joshhanley** [reply]
@nowhinjing-ever yeah @mauritskorse is correct, this already exists 🙂
